_pages/relays/nostrRelay.md

```diff
@@ -0,0 +1,18 @@
+---
+title: nostr_relay
+permalink: /nostr_relay/
+web: https://code.pobblelabs.org/fossil/nostr_relay/index
+github: 
+tags:
+- relay
+progLang: 
+license: 
+- Copyright
```

Comment:
Copyright is not a license.
`license:` is not an array.
[it](https://code.pobblelabs.org/fossil/nostr_relay/file?name=LICENSE.txt&ci=tip) looks like some copy-left license but not named, so let's leave it out.

```suggestion
license: 
```